src/galileo/logger/logger.py

```diff
@@ -533,7 +533,7 @@ def start_trace(
                 - String: "User query: What is the weather today?"
                 - Messages: `[Message(content="Hello", role=MessageRole.user)]`
         redacted_input: Optional[StepAllowedInputType]
-            Redacted input to the node.
+            Input that removes any sensitive information (redacted input).
```

Comment:
Apply this to all the `redacted_input` parameters for other methods on the logger.